### PR TITLE
h2 connection pools; suppress lbs deploy on release

### PR DESCRIPTION
**Pull request by kaiserpelagic**

### CODE REVIEW

File: core/src/main/scala/storage/hikari.scala [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import doobie.contrib.hikari.hikaritransactor._
+import scalaz.concurrent.Task
+
+object Hikari {
+
+  def build(db: DatabaseConfig): HikariTransactor[Task] = {
+    val trans = for {
+      xa <- HikariTransactor[Task](db.driver, db.connection, db.username.getOrElse(""), db.password.getOrElse(""))
+       _ <- xa.configure(hx => Task.delay(db.maxConnections.foreach(max => hx.setMaximumPoolSize(max))))
```
Comment (rossabaker) [inline core/src/main/scala/storage/hikari.scala]: What happens if this is not set?  Does hikari provide a reasonable default?

File: core/src/test/scala/TestStorage.scala [hunk 2]
```diff
@@ -16,16 +16,16 @@
...
   def dbConfig(testname: String): DatabaseConfig =
-    DatabaseConfig("org.h2.Driver", s"jdbc:h2:file:db/nelson.test.$testname.db;DATABASE_TO_UPPER=FALSE;", None, None)
+    DatabaseConfig("org.h2.Driver", s"jdbc:h2:file:db/nelson.test.$testname.db;DATABASE_TO_UPPER=FALSE;", None, None, None)
 
   def storage(testname: String) = {
-    val s = new nelson.storage.H2Storage(dbConfig(testname))
-    nelson.storage.run(s, StoreOp.migrate).run
-    s
+    val cfg = dbConfig(testname)
+    Migrate.migrate(cfg).run
```
Comment (rossabaker) [inline core/src/test/scala/TestStorage.scala]: I'm still not fond of this run at not-the-end-of-the-world, but it's no worse.

File: core/src/main/scala/storage/hikari.scala (hunk 1 above)
Comment (kaiserpelagic) [inline core/src/main/scala/storage/hikari.scala]: That's a good question, let me investigate.

Comment (kaiserpelagic) [inline core/src/main/scala/storage/hikari.scala]: default is 10, which is a very sane default.

File: core/src/test/scala/TestStorage.scala (hunk 2 above)
Comment (kaiserpelagic) [inline core/src/test/scala/TestStorage.scala]: agreed, was just trying to keep parity with what was already there

File: core/src/main/scala/storage/hikari.scala (hunk 1 above)
Comment (rossabaker) [inline core/src/main/scala/storage/hikari.scala]: I'm good with that.

Comment (timperrett) [inline core/src/main/scala/storage/hikari.scala]: 👍

**Pull request merged**